util/serial/src/lib.rs

```diff
@@ -200,3 +268,26 @@ mod json_u64_tests {
         assert_eq!(deserialized, the_struct);
     }
 }
+
+#[cfg(test)]
```

Comment:
This should use the same `cfg` condition
```suggestion
#[cfg(all(test, feature = "serde_with"))]
```